src/js/server/routes/apply.js

```diff
@@ -52,15 +52,42 @@ applyRouter.post('/form', auth.authenticate, applyFormUpload.single('cv'), (req,
       // Store the hacker information in the database
       const user = res.locals.user;
       const form = resultForm.data;
+      form.cv = req.file;
       const applicationID = crypto.randomBytes(64).toString('hex');
       database.Hacker.create({
+        // Personal
         firstName: user.first_name,
         lastName: user.last_name,
-        applicationID,
+        gender: user.gender,
+        dateOfBirth: user.date_of_birth,
         email: user.email,
         phoneNumber: user.phone_number,
-        inTeam: form.team_apply,
-        wantsTeam: form.team_placement,
+        // Education
+        institution: user.school.name,
+        studyLevel: user.level_of_study,
+        course: user.major,
+        // Logistics
+        shirtSize: user.shirt_size,
+        dietaryRestrictions: user.dietary_restrictions,
+        specialNeeds: user.special_needs,
```

Comment:
Not a blocker (we can look at this in another PR), but it might make more sense to create the user when we auth with MyMLH. Then we can store just user ID in a cookie and populate `req.user` with that.